src/TLuaInterpreter.cpp

```diff
@@ -2791,25 +2752,19 @@ int TLuaInterpreter::getExitStubs1(lua_State* L)
     // Previously threw a Lua error on non-existent room!
     TRoom* pR = host.mpMap->mpRoomDB->getRoom(roomId);
     if (!pR) {
-        lua_pushnil(L);
-        lua_pushfstring(L, "getExitStubs1: bad argument #1 value (number %d is not a valid room id).", roomId);
-        return 2;
-    } else {
-        QList<int> stubs = pR->exitStubs;
-        if (!stubs.empty()) {
-            lua_newtable(L);
-            for (int i = 0, total = stubs.size(); i < total; ++i) {
-                lua_pushnumber(L, i + 1);
-                lua_pushnumber(L, stubs.at(i));
-                lua_settable(L, -3);
-            }
-            return 1;
-        } else {
-            lua_pushnil(L);
-            lua_pushfstring(L, "getExitStubs1: no stubs in this room with id %d.", roomId);
-            return 2;
-        }
+        return warnArgumentValue(L, __func__, QStringLiteral("bad argument #1 value (number %1 is not a valid room id)").arg(roomId));
+    } 
+    QList<int> stubs = pR->exitStubs;
+    if (!stubs.empty()) {
```

Comment:
This has a logic error:
```suggestion
    if (stubs.empty()) {
```

Reply:
That is deliberate

Reply:
But it is wrong!

If there ARE stubs you are printing an error about there NOT being any stubs.

Reply:
My bad, you are correct. 
I changed the logic in some places, but did forget it here.